src/Equinor.ProCoSys.IPO.Infrastructure/IPOContext.cs

```diff
@@ -37,6 +38,30 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             base.OnModelCreating(modelBuilder);
             modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
             SetGlobalPlantFilter(modelBuilder);
+            SetIndexes(modelBuilder);
```

Comment:
Undo changes in this file and move index creation to existing ParticipantConfiguration and existing InvitationConfiguration